xmodule/capa/safe_exec/safe_exec.py

```diff
@@ -347,14 +347,14 @@ def normalize_error_message(emsg):
 
 
 def report_darklaunch_results(
-        *, slug,
+        *, limit_overrides_context, slug,
         globals_local, emsg_local, unexpected_exc_local,
         globals_remote, emsg_remote, unexpected_exc_remote,
 ):
     """Send telemetry for results of darklaunch."""
     can_compare_output = True
 
-    def report_arm(arm, globals_dict, emsg, unexpected_exception):
+    def report_arm(arm, emsg, unexpected_exception):
```

Comment:
Unrelated, but what is arm and could there be a docstring?

Reply:
Maybe I should know this, but what does "arm" mean? The values help a lot, but renaming to "foot" or "something" would be equally meaningful to me. This this a term I should know?

Reply:
I've pushed up a docstring that might help evoke that better.

Reply:
Thanks. Sometimes you learn stuff that everyone else knows I guess. :)